src/schema/objects/common_principles.yaml

```
@@ -87,6 +87,9 @@ label:
     for example, it is `rest` in `task-rest` following `task-<label>` specification.
     Note that labels MUST not collide when casing is ignored
     (see [Case collision intolerance](SPEC_ROOT/common-principles.md#case-collision-intolerance)).
+    Labels MUST be consistent across subjects and sessions and data types.
+    For example, an `acq` entity label `hires` used on the `anat` data of a given subject
+    MUST mean the same thing as `acq-hires` in `func` files of any other subject.
```

MUST is IMHO too strong of a word here since we do not even define the meaning of "mean" in this case -- is it  EXACTLY the same resolution? the same shimming? clearly it cannot be "the same everything" since then functional must be an anatomical ;)  Other example could be `_desc` for which we provision `descriptions.tsv` per each subject/session hence allowing for different "meaning"s (e.g. exact filtering parameters) for the same overall descriptor `filt`.

Hence

```suggestion
    Labels SHOULD be consistent across subjects and sessions and data types.
    For example, an `acq` entity label `hires` used on the `anat` data of a given subject
    SHOULD mean the same thing as `acq-hires` in `func` files of any other subject.
```

---

might be additional factor -- IIRC bids-validator would issue a WARNING whenever "same" (e.g. `_task-movie_run_1`) files across subjects have different "qualities" (e.g. length), not ERROR.  If changed to `MUST`, it then MUST issue an ERROR rendering some previously valid BIDS datasets invalid, breaking change.